LICENSE.md

```diff
@@ -0,0 +1,67 @@
+Portions of this repo are based on [Ingo Schommer's github-dashing](https://github.com/chillu/github-dashing).
+
+## Ingo Schommer's work
+
+[Ingo Schommer's github-dashing](https://github.com/chillu/github-dashing), and code we retain here from it, is released under the MIT License:
+
+```
+Copyright (c) 2014 Ingo Schommer
+
+Permission is hereby granted, free of charge, to any person obtaining
+a copy of this software and associated documentation files (the
+"Software"), to deal in the Software without restriction, including
+without limitation the rights to use, copy, modify, merge, publish,
+distribute, sublicense, and/or sell copies of the Software, and to
+permit persons to whom the Software is furnished to do so, subject to
+the following conditions:
+
+The above copyright notice and this permission notice shall be
+included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+```
+
+## Public domain
+
+18F's contributions to this project are covered by the following:
```

Comment:
this is cool, will definitely reference when I work on 18F projects that are forks of other codebases